sdk/notificationhubs/notification-hubs/src/utils/optionUtils.ts

```diff
@@ -1,9 +1,24 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT License.
 
-import { DirectSendNotificationOptions, SendNotificationOptions } from "../models/options.js";
+import {
+  BroadcastSendNotificationOptions,
+  DirectSendNotificationOptions,
+  SendNotificationOptions,
+} from "../models/options.js";
 import { objectHasProperty } from "@azure/core-util";
 
+/**
+ * Determines whether the options are of type SendNotificationOptions.
```

Comment:
`SendNotificationOptions` in doc comments should be `BroadcastSendNotificationOptions`?

Reply:
Fixed